app/helpers/proposals_helper.rb

```diff
@@ -131,4 +131,14 @@ def get_available_actions_text
   def display_submitter_name(submitter_id, user_hash = {})
     user_hash[submitter_id] || 'Pending'
   end
+
+  def proposal_published_notification_view_collections_link
+    if Rails.env.development? || Rails.env.test?
+      Rails.configuration.services.dig('earthdata', 'sit', 'cmr_root')
```

Comment:
I thought we talked about these pointing to the local cmr.

Reply:
Okay I think I fixed this. Let me know if it doesnt work